workflows/methylation/methylation-cohort.wdl

```diff
@@ -2,6 +2,7 @@ version 1.1
 
 workflow methylation_cohort {
     meta {
+        name: "Methylation Cohort"
```

Comment:
This is redundant (if we keep the plans to "capital case" to determine `name` automatically)

Reply:
Honestly I removed the redundant because I thought that was what you were asking for 😅 lol. I don't think it's a big deal either way, but the consistent thing would be to remove it.